.github/workflows/scan.yml

```diff
@@ -0,0 +1,60 @@
+name: Scan
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - "main"
+      - "rc"
+      - "hotfix-rc"
+  pull_request:
+
+permissions: read-all
+
+jobs:
+  sast:
+    name: SAST scan
+    runs-on: ubuntu-22.04
```

Comment:
Can we add a condition to exclude forks?

```yml
if: "! github.event.pull_request.head.repo.fork"
```

Reply:
We'll eventually want this running in all cases. See chat but we can at some point in time look at a total shift here to support community contributions. I think this has to be status quo for the moment.